docs/_sass/_main.scss

```diff
@@ -986,3 +986,11 @@ button {
 .hidden {
     display: none;
 }
+
+.expensify-text-svg {
```

Comment:
NAB: Maybe we use a different classnames for this (and the one below), like:
- `.expensifyhelp-logo__expensify`
- `.expensifyhelp-logo__help`

Reply:
Updated